wallet-new/src/Cardano/Wallet/WalletLayer/Kernel/Conv.hs

```diff
@@ -233,6 +244,11 @@ instance Buildable InvalidRedemptionCode where
 instance Show InvalidRedemptionCode where
     show = formatToString build
 
+instance Arbitrary InvalidRedemptionCode where
+    arbitrary = oneof [ InvalidRedemptionCodeInvalidBase64 <$> arbitrary
+                      , InvalidRedemptionCodeCryptoError <$> arbitrary
```

Comment:
Looking into this. Think we should have an arbitrary instance for `CryptoError`, even if they are turned into strings (which I'm also beginning to question more).

This is just using the `arbitrary` of `String`.

Reply:
Done. Pushing soon. Also, realised `CryptoError` has no payload, so `Text` seems fine to me.

Reply:
Sorry, I was busy and didn't answer. To answer you: I'd rather see a newtype than an orphan instance :) 
You could use a newtype locally to wrap and unwrap the error in order to define the Arbitrary instance.

Reply:
Oh no, sorry, was mostly talking out loudly, and also formulated myself carelessly.

I simply wrote a `genCryptoError :: Gen CryptoError` helper. 

(2456a74ef2fa4c59584187938552f5219cca9976, rebasing soon)